index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Fork Me! FCC: Test Suite Template</title>
+  <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+  <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+  <header>
+    <h1 id='title'> “Football Survey Form”</h1>
+    <p id='description'>Choose the best player(s) in Africa</p>
+  </header>
+  <form id="survey-form">
+    <label id="name-label">Name</label>
+    <input id="name" type="text" required placeholder="input your name">
+
+    <label id="email-label">Email</label>
+    <input id="email" type="email" required placeholder="input your email">
+
+    <label id="number-label">Age</label>
+    <input id="number" type="number" required max="70" min="18" placeholder="age">
+    <select id="dropdown">
```

Comment:
Too many ids are used, please try to use fewer ids and more classes.
classes make your code clean and easy to read